interfaces/builtin/network_manager_test.go

```diff
@@ -149,7 +156,14 @@ func (s *NetworkManagerInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	snippet, err = s.iface.PermanentSlotSnippet(s.slot, interfaces.SecurityDBus)
 	c.Assert(err, IsNil)
 	c.Assert(snippet, Not(IsNil))
-	snippet, err = s.iface.PermanentSlotSnippet(s.slot, interfaces.SecuritySecComp)
+}
+
+func (s *NetworkManagerInterfaceSuite) TestSecCompPermanentSlot(c *C) {
+	seccompSpec := &seccomp.Specification{}
+	err := seccompSpec.AddPermanentSlot(s.iface, s.slot)
 	c.Assert(err, IsNil)
-	c.Assert(snippet, Not(IsNil))
+	snippets := seccompSpec.Snippets()
+	c.Assert(len(snippets), Equals, 1)
+	c.Assert(len(snippets["snap.network-manager.nm"]), Equals, 1)
+	c.Check(string(snippets["snap.network-manager.nm"][0]), testutil.Contains, "listen\n")
```

Comment:
(nitpick²) maybe we test for sethostname here as that is slightly more unique to networkmanager than listen which is very generic. but really hardcore nitpick, feel free to ignore :)